Auth/class.LDAPAuthenticator.php

```diff
@@ -190,11 +203,13 @@ public function login($username, $password)
         }
         $user = $user[0];
         $server->unbind();
+
         $ret = $server->bind($user->dn, $password);
         if($ret !== false)
         {
-            return array('res'=>true, 'extended'=>$user); 
+            return array('res'=>true, 'extended'=>$user);
         }
+        $server->unbind();
```

Comment:
we bind as the user her just to authenticate their password.. yes?
for most other functions we unbind or close the connection at the end of the function..
i thought it made sense to do so here as well